### ISSUE TITLE
Ablation Study

### ISSUE THREAD

**Issue by xpngzhng** [thuyngch/ATSS-EfficientDet-PyTorch#9]
Hi @thuyngch 
thank you for your sharing
I see your config file for efficientdet, ATSS is used, SyncBN is used and cosine learning rate is used, have you done training experiment replacing ATSS with original RetinaHead and using step learning rate? How much will the bbox mAP changes? 
It seem that SyncBN dramatically improves mAP (from other issue https://github.com/thuyngch/ATSS-EfficientDet-PyTorch/issues/7)
